### PR TITLE
[oddcontrast-tool] OddContrast Tool

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
/tools/
/oddcontrast/

### CODE REVIEW

**Comment by SondraE** [comment]
[Add OddContrast to OddSite developer tools section](https://trello.com/c/YQxsckMy/218-add-oddcontrast-to-oddsite-developer-tools-section)

File: content/index.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ tools:
   - /susy/
   - /accoutrement/
   - /herman/
+  - /oddcontrast/
```
Comment (jgerigmeyer) [inline content/index.md:42]: @SondraE What do you think about adding it to the homepage like this? Feel free to remove this if you prefer.

**Review by jgerigmeyer — APPROVED** [review]

File: content/tools/oddcontrast.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  src='projects/out-of-gamut.jpg',
+  alt='display-p3 light yellow color showing hexidecimal color
+    outside the sRGB gamut and the hsl color outside the hsl gamut'
+) }}
+
+## Contrast Checker for Accessibility
+
+OddContrast checks the contrast ratio between two colors and displays pass or
+fail information based on the WCAG 2 accessibility standards for AA and AAA in
+normal and large font sizes. If you need a reminder of contrast ratio standard
```
Comment (stacyk) [inline content/tools/oddcontrast.md:45]:
```suggestion
normal and large font sizes. If you need a reminder of the contrast ratio requirements
```

File: content/tools/oddcontrast.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  featuring newer color spaces like
+  Oklch, Oklab, and P3.
+  It's also a color contrast checker
+  to help web designers meet
+  WCAG 2 accessibility standards.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+## Support for New Color Spaces
```
Comment (stacyk) [inline content/tools/oddcontrast.md:27]:
```suggestion
## Support for Newer Color Formats and Spaces
```

**Review by stacyk — COMMENTED** [review]
Great work @SondraE! I left a few clean up items here based on the terminology from https://drafts.csswg.org/css-color-4/ and a few more loosely opinionated/optional suggestions in phrasing.

File: content/tools/oddcontrast.md (hunk 2 above)
Comment (stacyk) [reply content/tools/oddcontrast.md:44]: @SondraE wanted to double check with you on this one... I don't mind if you don't take the suggestions, but in the phrase "If you need a reminder of contrast ratio standard"  shouldn't that last word either be standards or the phrase would need a "the" before the word contrast? Or am I reading this out of context?

File: content/_includes/logos/oddcontrast.svg [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<svg
+  data-logo="oddcontrast"
+  xmlns="http://www.w3.org/2000/svg"
+  aria-hidden="true"
+  focusable="false"
+  width="511.81px"
+  height="63.45px"
```
Comment (jgerigmeyer) [inline content/_includes/logos/oddcontrast.svg:7]: @stacyk I made up these height + width values -- not sure if you want to adjust?

File: content/tools/oddcontrast.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+link to learn more about ratio requirements.
+
+{{ embed.img(
+  src='projects/color-ratio.jpg',
+  alt='showing the red and yellow colors at 4.37:1 contrast ratio
+    failing for AA Normal, AAA Normal, and AAA Large font sizes,
+    but passing for AA Large font size'
+) }}
+
+## What New Features Should OddContrast Have?
```
[the remark this answers is not in the document]
Comment (SondraE) [reply content/tools/oddcontrast.md:58]: Added

File: content/_includes/logos/oddcontrast.svg (hunk 4 above)
Comment (stacyk) [reply content/_includes/logos/oddcontrast.svg:7]: @jgerigmeyer They look fine to me. As long as it matches the aspect ratio in the viewbox (or exactly) then I think it is fine. We don't need the px unit there, but it doesn't hurt anything.

**Review by stacyk — APPROVED** [review]
LGTM!

**Pull request merged**